source/grid/manifold_lib.cc

```diff
@@ -1805,9 +1805,8 @@ namespace
         // weights are set outside and coincide with the flat manifold case)
 
         double weights_lines[GeometryInfo<3>::lines_per_cell];
-        for (unsigned int line = 0; line < GeometryInfo<3>::lines_per_cell;
-             ++line)
-          weights_lines[line] = 0;
+        for (double &weights_line : weights_lines)
+          weights_line = 0;
```

Comment:
If you are willing, we could do the same
```cpp
std::array<double, GeometryInfo<3>::lines_per_cell> weights_lines{};
```
here to get rid of the loop.

Reply:
Sure.